### ISSUE TITLE
[Feature Request]: Make JASP and NVDA for Visually Impaired 

### ISSUE THREAD

**Issue by statsNERD23** [jasp-stats/jasp-issues#1865]
### Description

Please make JASP compatible with NVDA for people 

### Purpose

Accessibility 

### Is your feature request related to a problem?

People with visual impairment cannot use JASP

### Describe the solution you would like

Make JASP compatible with with NVDA (a free screen reader)

### Describe alternatives that you have considered

other screen readers, like JAWS

**Comment by SwanseaUniversityTransscriptionCentre** [comment]
Is there any update on this? We are supporting a visually impaired student who uses a screen reader and will need to use JASP next year.

**Comment by tomtomme** [comment]
closing, this is a duplicate of https://github.com/jasp-stats/jasp-issues/issues/1783
I have no updates on this yet, sorry. But the request is surely not forgotten. We recently added ALT-Navigation mode which was a step in the right direction.
